styles.css

```diff
@@ -512,32 +575,173 @@ footer {
   height: 40px;
 }
 
-@media only screen and (min-width: 768px) {
-  .heading {
-    margin-left: 154px;
-  }
+.home-icon {
+  display: block;
+  margin: 0 auto;
+  padding-bottom: 10px;
+}
 
-  .nav-ul {
-   display: flex;
-   flex-direction: row;
-   width: 0px;
-   height: 0;
-   text-align: center;
-   align-items: center;
-   justify-content: end;
-   position: relative;
-   padding-top:2px;
-   margin: 0;
-  }
+.modal {
+  display: none;
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  background-color: #3c3a39;
+}
+
+.modal-content {
+  padding: 25px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 25px;
+}
+
+.close-modal {
+  width: 100%;
+  text-align: end;
+}
+
+.modal-links {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  text-decoration: none;
+  gap: 20px;
+  color: #ffff;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  border-bottom: 2px solid #6f6c6b;
+}
+
+.popup-content > ul > li {
+  color: #3c3a39;
+  border: 1px solid #979493;
+  padding: 12px;
+}
+
+.modal-links > li:hover {
+  transition: 0.3s ease;
+  transform: scale(1.1);
+  transform: skew(10deg);
+}
+
+.popup-container {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  flex-direction: row;
+  left: 0;
+  top: 0;
+  z-index: 1;
+  background: rgba(151, 148, 147, 0.8);
+}
+
+.popupLink {
+  text-decoration: none;
+  color: white;
+}
+
+article.popup-content > * {
+  margin: 0;
+  padding: 0;
+  position: static;
+}
+
+.popup-content {
+  position: relative;
+  margin: 30px 16px;
+  height: 718px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  gap: 10px;
+  padding: 15px;
+  background-color: #fff;
+}
```

Comment:
Kindly fix the description text in your first popup window .
![rev1](https://user-images.githubusercontent.com/53347966/175518781-30f81980-e330-48a0-8d44-601ce512bf20.JPG)